scripts/mac/kandji/v1/zluri-mac-hide-icon-script.sh

```
@@ -0,0 +1,84 @@
+#!/bin/bash
+CURRENT_USER=$( echo "show State:/Users/ConsoleUser" | scutil | awk '/Name :/ && ! /loginwindow/ { print $3 }' )
+HOMEDIR=$(/usr/bin/dscl . -read /Users/"$CURRENT_USER" NFSHomeDirectory | /usr/bin/cut -d' ' -f2)
+ORG_TOKEN=<orgToken> # needs to be added by the customer
```

Shellcheck (0.10.0)

[error] Couldn't parse this variable assignment. Fix to allow more checks.

(SC1073)